src/ipc/ipc4/handler.c

```diff
@@ -250,50 +250,48 @@ static int set_pipeline_state(uint32_t id, uint32_t cmd, bool *delayed, uint32_t
 
 	switch (cmd) {
 	case SOF_IPC4_PIPELINE_STATE_RUNNING:
-		/* nothing to do if the pipeline is already running */
```

Comment:
nit-pick: can we keep the inner cases in the same order for the all outer switch/cases?

Reply:
@ranj063 sorry, which outer switch-case? The outer one is over commands, the inner one is over states